sparql-ns.ttl

```diff
@@ -242,17 +242,17 @@ sparql:strlang rdf:type sparql:Function ;
     .
 
 sparql:strlangdir rdf:type sparql:Function ;
-    rdfs:comment "" ;
+    rdfs:comment "This function creates an RDF literal with language tag and initial text direction." ;
```

Comment:
Related descriptions/comments refer to `initial base direction` instead of `initial text direction`. We should be consistent.